addons/udes_stock/models/stock_picking.py

```diff
@@ -237,9 +239,11 @@ def _compute_state(self):
         """ Prevent pickings to be in state assigned when not able to handle
             partials, so they should remain in state waiting or confirmed until
             they are fully assigned.
+
+            Add a context variable to the can_handle_partials call
```

Comment:
Would be good to improve this explanation, isn't this so that in can_handle_partials() we can distinguish if we are in the compute state function?

Have we considered using a parameter instead of context variable?

Do we want to call the field `computing_state` instead?